Installation/CHANGES.md

```diff
@@ -3,6 +3,11 @@
 
 ## [Release 6.1](https://github.com/CGAL/cgal/releases/tag/v6.1)
 
+### [Polygon Mesh Processing](https://doc.cgal.org/6.1/Manual/packages.html#PkgPolygonMeshProcessing)
+- new implementation of `CGAL::Polygon_mesh_processing::clip()` with a plane as clipper that is much faster and is now able to handle non-triangulated surface meshes.
```

Comment:
document the breaking change on return type? (not just to say the return type changed, but I guess now you guarantee that the output is manifold?)

Reply:
I'll restore the bool to keep the API consistent with other overloads